docs/src/content/ignition/docs/getting-started/index.md

```diff
@@ -32,30 +32,36 @@ To install Hardhat Ignition in an existing Hardhat project, you will need:
 
 You can also follow [Hardhat's Quick Start guide](../../../hardhat-runner/docs/getting-started/index.md) to create a new project from scratch to follow this guide.
 
+:::tip
+
+If you prefer to use **Viem** instead of **ethers**, check out the [Viem guide](../../../ignition/docs/guides/viem.md) for installation instructions.
+
+:::
+
 Once you have a Hardhat project ready, open a terminal in its root directory, and run:
 
 ::::tabsgroup{options="npm 7+,npm 6,yarn"}
 
 :::tab{value="npm 7+"}
 
 ```sh
-npm install --save-dev @nomicfoundation/hardhat-ignition
+npm install --save-dev @nomicfoundation/hardhat-ignition-ethers
 ```
 
 :::
 
 :::tab{value="npm 6"}
 
 ```sh
-npm install --save-dev @nomicfoundation/hardhat-ignition
```

Comment:
This should include the peer dependencies (transitively all of them).

I think this may be present in the other guides too.

Reply:
I have updated this npm 6/yarn call to include the transitive peer dependencies of `@nomicfoundation/hardhat-ignition-ethers` and done the same for `@nomicfoundation/hardhat-ignition-viem` in the viem guide.

I will do a double check of the manual install for them in the morning as part manually testing the release branch.